### PR TITLE
Add cmake

### PR DESCRIPTION
**Pull request by groutr**
"CMake is an open-source, cross-platform family of tools designed to build, test and package software."

From https://github.com/conda-forge/staged-recipes/pull/108

Since I'm not sure how to bootstrap cmake on windows, we simply repackage the official cmake windows binaries for now.  Everything is built from source on unix platforms.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@groutr you can add `[skip appveyor]` in your commit message to prevent AppVeyor from starting.

File: recipes/cmake/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  version: 3.5.0
+
+source:
+  url: https://cmake.org/files/v3.5/cmake-3.5.0.tar.gz  # [not win]
+  fn: cmake-3.5.0.tar.gz # [not win]
+  
+  url: https://cmake.org/files/v3.5/cmake-3.5.0-win32-x86.zip  # [win]
+  fn: cmake-3.5.0-win32-x86.zip # [win]
+
+
```
Comment (msarahan) [inline recipes/cmake/meta.yaml]: extra space here

File: recipes/cmake/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+about:
+  home: http://www.cmake.org/
+  license:  3-clause BSD
+  license_family: BSD
+  summary: CMake is an extensible, open-source system that manages the build process
+
+extra:
+  recipe-maintainers:
+    - groutr
+    - ocefpaf
```
Comment (msarahan) [inline recipes/cmake/meta.yaml]: Add me if you want.

File: recipes/cmake/meta.yaml (hunk 1 above)
Comment (groutr) [inline recipes/cmake/meta.yaml]: Does the extra space make a functional difference?

Comment (msarahan) [inline recipes/cmake/meta.yaml]: Nope.  Just nitpicking.  Feel free to ignore, or store up revenge for the next recipe of mine you review.

Comment (groutr) [inline recipes/cmake/meta.yaml]: I was curious because somebody mentioned a recipe linter.

**Comment by groutr** [comment]
Thanks @msarahan and @ocefpaf for the feedback.

**Comment by ocefpaf** [comment]
Awesome @groutr!  Thanks!!

**Comment by pelson** [comment]
Great stuff @groutr. Thanks for merging @ocefpaf. Definitely agree with the smaller PR approach in this case. :+1:

**Comment by jakirkham** [comment]
@bradking @billhoffman @steveire @neundorf @clintonstimpson, just letting all of you know that we have added `cmake` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ( [`conda install -c conda-forge cmake`]( http://anaconda.org/conda-forge/cmake ) ). The maintenance burden here is light as all releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this feedstock (repo)]( https://github.com/conda-forge/cmake-feedstock ). If you have any interest in being added as a maintainer (useful for making new releases for instance), we would be more than happy to add any of you to the maintainers list. Thanks for your contributions to this wonderful build tool.

**Pull request merged**